src/gatling/simulations/scenarios/Solicitor_Bails.scala

```diff
@@ -0,0 +1,1383 @@
+package scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import utils.{Common, Environment, Headers}
+
+/*======================================================================================
+* Create a new Bail application as a professional user (e.g. Legal Rep)
+======================================================================================*/
+
+object Solicitor_Bails {
+
+  val BaseURL = Environment.baseURL
+
+  val MinThinkTime = Environment.minThinkTime
+  val MaxThinkTime = Environment.maxThinkTime
+
+  val CreateBailApplication =
+
+
+    /*======================================================================================
+    * Click the Create Case link
+    ======================================================================================*/
+
+    group("XUI_Bails_030_CreateCase") {
+
+      exec(_.setAll(
+        "BailsApplicantFirstName" -> ("Bails" + Common.randomString(5)),
+        "BailsApplicantLastName" -> ("Test" + Common.randomString(5)),
+        "BailsFinancialFirstName" -> ("App" + Common.randomString(5)),
+        "BailsFinancialLastName" -> ("Test" + Common.randomString(5)),
+        "BailsLegalRepFirstName" -> ("Legal" + Common.randomString(5)),
+        "BailsJudgeName" -> ("Judge" + Common.randomString(5)),
+        "BailsAppDobDay" -> Common.getDay(),
+        "BailsAppDobMonth" -> Common.getMonth(),
+        "BailsAppDobYear" -> Common.getDobYear(),
+        "BailsHomeOfficeRefNum" -> (Common.randomNumber(7)),
+        "BailsNomsNumber" -> (Common.randomNumber(7)),
+        "BailsArrivedYear" -> Common.getDobYearChild(),
+        "BailsAppPhoneNumber" -> ("07" + Common.randomNumber(9)),
+        "BailsSupporterPhoneNumber" -> ("07" + Common.randomNumber(9)),
+        "BailsSupporterEmail" -> (Common.randomString(7) + "@gmail.com"),
+        "BailsSupporterDobDay" -> Common.getDay(),
+        "BailsSupporterDobMonth" -> Common.getMonth(),
+        "BailsSupporterDobYear" -> Common.getDobYear(),
+        "BailsSupporterRelationship" -> (Common.randomString(7)),
+        "BailsSupporterOccupation" -> (Common.randomString(7)),
+        "BailsPassportNumber" -> (Common.randomNumber(7)),
+        "BailsEvidenceDescription" -> (Common.randomString(7) + Common.randomString(6) + Common.randomString(5)),
+        "BailsSupporterImmigration" -> (Common.randomString(7)),
+        "BailsLegalRepReference" -> (Common.randomString(7) + Common.randomString(7)),
+        "BailsConditionsAppearance" -> (Common.randomString(7) + "" +  Common.randomString(7) +  Common.randomString(7) +  Common.randomString(7)),
```

Comment:
These ' + "" ' don't do anything...

Reply:
Yeah agreed I knew what I wanted to do but didn't do it right (this has now been fixed)